recipes/atomgpt/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "atomgpt" %}
+{% set version = "2024.6.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/atomgpt/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b4620ec1b090c10fb76acc0d91342b5edee9e8de242171e02d3a04173fc8f4c2
+
+build:
+  noarch: python 
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools 
+    - wheel
+    - pip
+  run:
+    - python >=3.10
+    - numpy >=1.19.5
+    - scipy >=1.6.1
+    - jarvis-tools >=2021.07.19
+    - pytorch >=2.0.0
+    - mpmath >=1.3.0
+    # - pytorch-cuda >=12.1
+    - trl
+    - peft
+    - accelerate 
+    - bitsandbytes
+    - triton
+    - sentencepiece
+    - transformers
+    - xformers 
+    - spglib >=2.0.2
+    - scikit-learn >=0.22.2
+    - matplotlib-base >=3.4.1
+    - tqdm >=4.60.0
+    - pandas >=1.2.3
+    - pydantic >=1.8.1
+    - pydantic-settings
+    - flake8 >=3.9.1
+    - pycodestyle >=2.7.0
+    - pydocstyle >=6.0.0
+    - pyparsing >=2.2.1,<3
+
+
+test:
+  imports:
+    - atomgpt
+  commands:
+    - pip check
+    # - train_alignn.py -h
```

Comment:
Why is this test commented out?

Reply:
`train_alignn.py` came from another conda recipe template I was using, removed this line now